src/views/studio/EditVideoSheet/EditVideoForm/EditVideoForm.tsx

```diff
@@ -171,11 +173,12 @@ export const EditVideoForm: React.FC<EditVideoFormProps> = ({
 
   // with react-hook-form v7 it's possible to call watch((data) => update()), we should use that instead when we upgrade
   const handleFormChange = () => {
+    const data = getValues()
     if (!selectedVideoTab?.isDraft) {
-      return
+      updateTabData(data)
+    } else {
+      debouncedDraftSave.current(selectedVideoTab, data, addDraft, updateDraft, updateSelectedVideoTab)
```

Comment:
We probably should update the debounced fn name, because now it doesn't save drafts only

Reply:
but it does only save drafts, no?

Reply:
I'm actually not sure what I meant here, the fn still only saves drafts.. Let's ignore this comment :P